.github/workflows/pr.yaml

```
@@ -104,6 +104,9 @@ jobs:
     if: fromJSON(needs.changed-files.outputs.changed_file_groups).test_cpp
     with:
       build_type: pull-request
+      # Filtering out 11.4 runs until 11.4 issues are resolved
+      # See https://github.com/rapidsai/build-planning/issues/164
+      matrix_filter: map(select(.CUDA_VER != "11.4.3"))
```

```suggestion
      matrix_filter: map(select(! .CUDA_VER.startswith("11.4")))
```

Would you consider something like this that changes the condition from "not identical to 11.4.3" to "does not being with 11.4"?

That'd avoid this breaking in the (I know, probably unlikely) instance where we switch to a different patch version of CUDA 11.4 or where CUDA_VER is changed from `11.4.3` to just `11.4`.

`jq` definitely does have `startswith()` (https://jqlang.org/manual/#startswith), but not sure if I have the syntax exactly right here.

---

I can come back and update it to something like

```
(.CUDA_VER | startswith("11.4")) | not)
```

if this is going to continue in-place, but my hope is that this is a very short-term filter

---

ok no prob